src/spring/HISTORY.md

```diff
@@ -2,4 +2,9 @@ Release History
 ===============
 1.0.0
 ---
-* Initialize extension "Spring" to manage Azure Spring Apps resources.
\ No newline at end of file
+* Initialize extension "Spring" to manage Azure Spring Apps resources.
+
+1.1.0
+---
+* New argument `az spring create` has new argument "--ingress-read-timeout" to set ingress read timeout when create Azure Spring App.
+* New argument `az spring update` has new argument "--ingress-read-timeout" to update ingress read timeout for Azure Spring App.
```

Comment:
Please write the history notes of the new version at the top of the old version

Reply:
Thanks, Updated/